### ISSUE TITLE
Missing method exception in GetEntityName in AuditDbContext.Core.cs

### ISSUE THREAD

**Issue by john-mathews** [thepirat000/Audit.NET#25]
entityType.SqlServer() method is not found when trying to get entity name

`System.MissingMethodException: Method not found: 'Microsoft.EntityFrameworkCore.Metadata.IRelationalEntityTypeAnnotations Microsoft.EntityFrameworkCore.SqlServerMetadataExtensions.SqlServer(Microsoft.EntityFrameworkCore.Metadata.IEntityType)'.
   at Audit.EntityFramework.AuditDbContext.GetEntityName(IEntityType entityType)
   at Audit.EntityFramework.AuditDbContext.CreateAuditEvent(Boolean includeEntities, AuditOptionMode mode)
   at Audit.EntityFramework.AuditDbContext.SaveChanges()`

**Comment by thepirat000** [comment]
can you share the dependencies on your project.json? I was not able to reproduce, which version of `Microsoft.EntityFrameworkCore.SqlServer` are you using?

**Comment by john-mathews** [comment]
`  "dependencies": {
    "NETStandard.Library": "1.6.1",
    "Microsoft.EntityFrameworkCore": "1.1.0",
    "Microsoft.EntityFrameworkCore.SqlServer":"1.1.0",
    "System.Data.SqlClient": "4.3.0",
    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.1.0",
    "Microsoft.Extensions.Configuration.FileExtensions": "1.1.0",
    "Microsoft.Extensions.Configuration.Json": "1.1.0",
    "Microsoft.Extensions.Configuration.CommandLine": " 1.1.0",
    "Audit.EntityFramework": "5.2.0",
    "Audit.NET.SqlServer": "5.2.0",
    "Audit.NET": "5.2.0"
  },`

**Comment by john-mathews** [comment]
"dependencies": {

    "NETStandard.Library": "1.6.1",

    "Microsoft.EntityFrameworkCore": "1.1.0",

    "Microsoft.EntityFrameworkCore.SqlServer":"1.1.0",

    "System.Data.SqlClient": "4.3.0",

    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.1.0",

    "Microsoft.Extensions.Configuration.FileExtensions": "1.1.0",

    "Microsoft.Extensions.Configuration.Json": "1.1.0",

    "Microsoft.Extensions.Configuration.CommandLine": " 1.1.0",

    "Audit.EntityFramework": "5.2.0",

    "Audit.NET.SqlServer": "5.2.0",

    "Audit.NET": "5.2.0"

  },

**Comment by thepirat000** [comment]
The error is because you are pointing to a newer version of the `Microsoft.EntityFrameworkCore.SqlServer` package, the supported version is 1.0.1 but you're pointing to 1.1.0.

It should work like this:

```javascript
  "dependencies": {
    "NETStandard.Library": "1.6.1",
    "Microsoft.EntityFrameworkCore": "1.0.1",
    "Microsoft.EntityFrameworkCore.SqlServer": "1.0.1", 
    "System.Data.SqlClient": "4.3.0",
    "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0",
    "Microsoft.Extensions.Configuration.FileExtensions": "1.0.0",
    "Microsoft.Extensions.Configuration.Json": "1.0.0",
    "Microsoft.Extensions.Configuration.CommandLine": "1.0.0",
    "Audit.EntityFramework": "5.2.0",
    "Audit.NET.SqlServer": "5.2.0",
    "Audit.NET": "5.2.0"
  },
```

**Comment by thepirat000** [comment]
This was fixed on version 6.0.0. (Now targeting NETStandard.Library 1.6.1)
Please re-open if you still have problems with new version.
